tests/scripts/analyze_outcomes.py

```diff
@@ -344,6 +344,12 @@ def do_analyze_driver_vs_reference(results: Results, outcomes: Outcomes, args) -
                     'Key ASN1 (Encrypted key PKCS5, trailing garbage data)',
                     re.compile(r'Parse (RSA|EC) Key .*\(.* ([Ee]ncrypted|password).*\)'),
                 ],
+                # TLS password protected server/client is not supported so far.
```

Comment:
There's no “password protected server/client”: what's password-protected is a key (or more precisely the encoding of a key). The part that isn't supported isn't related to TLS. The offending tests use TLS, but it's the fact that they use a password-protected key file that is problematic. The reason is the same as in `test_suite_pkparse`, which directly tests the same code that loads password-protected keys.

We could use the same comment as with `test_suite_pkparse` since it's the same reason.